docs/user/customization.md

```diff
@@ -0,0 +1,122 @@
+# Cluster Customization
+
+The OpenShift Installer allows for several different levels of customization. It is important to understand how and why each of these levels is exposed and the ramifications of making changes at each of these levels. This guide will attempt to walk through each of them and provide examples for why an administrator may want to make customizations.
+
+## Customization Levels
+
+The various customization that the installer enables can be broken into four major levels: OpenShift, Kubernetes, Platform, and OS. These four levels are rough abstraction layers (OpenShift being the highest layer and OS being the lowest) and fall into either the managed or unmanaged buckets. The levels within the managed bucket (OpenShift and Platform) encompass customization that is safe to perform - installation and automatic updates will succeed regardless of the changes made (to a reasonable degree). The levels within the unmanaged bucket (Kubernetes and OS) encompass customization that is not necessarily safe - after introducing changes, installation and automatic updates may not succeed.
+
+### OpenShift Customization
+
+The most simple customization is exposed by the installer as an interactive series of prompts. These prompts are required and represent the lowest degree of customization. They are needed in order to get a running OpenShift cluster, but they aren't enough to get anything other than a vanilla deployment out of the box. Further customization is possible once the cluster has been provisioned, but isn't covered in this document as it is a "Day 2" operation.
```

Comment:
"the lowest decree of customization" -> "high-level customization"?

Link over to [here][1] (or something else in that repo?) for day-2 discussion?  It would be great to have a landing page for "[things you might want to configure next once the installer exits][2]".

[1]: https://github.com/openshift/training/blob/master/docs/03-explore.md#understanding-openshift
[2]: https://github.com/openshift/installer/pull/771#discussion_r238002745